azure-pipelines.yml

```diff
@@ -149,9 +146,13 @@ stages:
     displayName: "kvmtest"
     timeoutInMinutes: 240
     steps:
+    - task: DownloadPipelineArtifact@2
+      inputs:
+        artifact: sonic-buildimage.kvm
```

Comment:
i am not sure it will download the artifact from the current job?

Reply:
I learned on https://docs.microsoft.com/en-us/azure/devops/pipelines/artifacts/pipeline-artifacts?view=azure-devops&tabs=yaml

> Pipeline artifacts provide a way to share files between stages in a pipeline or between different pipelines.

In this case, we share files between stages.